src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -67,6 +67,14 @@
                 localStorage.setItem(profilerStorageKey + name, value);
             },
 
+            addEvent = function (element, eventName, callback) {
+                if (document.all) {
```

Comment:
> Using the method .attachEvent() in IE, In normal browsers, using .addEventListener, the var this points to the element, while in IE it points to the window object.

I've pushed changes for `this` usage. Now we have lots of boilerplate code for cross-browser event handling :smile: